### ISSUE TITLE
Add python container

### ISSUE THREAD

**Issue by mikeerickson** [laradock/laradock#768]
Hey Gang

Has anybody added support for `Python` container (I specifically need it available in the workspace and mysql containers).   If not, please let me know and I will take care of it.

Specifically need `Python 2.7`, but if I create container I will also expose `Python 3.x` support

**Comment by mikeerickson** [comment]
Integrate `pip` as well when performing this task

https://github.com/laradock/laradock/issues/791

**Comment by dtunes** [comment]
Hi, just curious what's the use case for a python container, supervisord?

**Comment by mikeerickson** [comment]
I use http://mycli.net/ which is written in python, thus requires python to exist :-) 

It is quite an edge case, thus reason I have just thrown it out there to see if it sticks.  I can surely install it on an as needed basis.  I was in a situation where I was having to kill / recreate my containers so frequently, it was frustrating.

However, now that my workflow has stabilized, this is not as "important".   If I have to recreate the mysql container, I just run a bash script to install the extra stuff I use.

At some point, I will readdress this issue, but for now, I am just moving forward with what is on by default.

**Comment by hounded** [comment]
https://github.com/laradock/laradock/pull/1077

**Comment by Mahmoudz** [comment]
@mikeerickson why closing your issue! I guess it's a good suggestion to have a Python container.

**Comment by mikeerickson** [comment]
@Mahmoudz I am not quite sure why I closed it, surely something that is still a good addition (as you have stated).

**Comment by marcelogarbin** [comment]
@mikeerickson any news?

**Comment by jarodsmk** [comment]
I too have a need for this, although with recent developments and the announcement of the EOL of Python 2.7 it might be good to get other versions working too.

I have a Laravel project which includes some more advanced UI features, and so we utilize the `node-gyp` and `node-sass` packages (which needs python to build) 

**TLDR**

+1 :)

**Comment by filips123** [comment]
Python 2.7 is EOL for more than 6 months. So is it now possible to also support Python 3? Could this be done in same way as #1077 except with `WORKSPACE_INSTALL_PYTHON3` environment variable and Python 3 packages?

**Comment by Mahmoudz** [comment]
Closing as stale — no meaningful activity in 5+ years. If this is still relevant on current Laradock, please open a fresh issue describing the use case with your current setup, or submit a PR. Thanks for the original suggestion.
